documentation/man/features/build.rst

```diff
@@ -76,6 +77,11 @@ OPTIONS
   keeps the kernel configuration. This option considers whether the user
   is using or not kw env.
 
+-C, \--distclean:
+  The distclean option will delete everything that make clean does and
+  remove the configuration. Like `--clean` This option also considers
+  whether or not the user is using kw env.
```

Comment:
To make this description more valuable, make it clear that under the wood, we are calling make distclean. Also, explain that this command will reset the kernel tree to its default; in other words, it will remove any configuration file and script output.